<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1041</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="calculateButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>calculate</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>330</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="inputEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>351</width>
      <height>194</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>output:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="outputEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>add state</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>518</width>
      <height>200</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>population:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="populationEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>GDP:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="GDPEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>first letter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstLetterEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>in war:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="warEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nameEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>type texes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="firstTaxEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="secondTexEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="thirdTaxEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>user input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="structEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>bin user input:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="newstructEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="addSentenceButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>500</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>279</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>result:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="arrayResultEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="calculateArrayButton">
       <property name="text">
        <string>calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>231</width>
      <height>28</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>element:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="elemntEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="addElemenButton">
       <property name="text">
        <string>add element</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>570</y>
      <width>373</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>number of element:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="numberOfArrayEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="createArrayButton">
       <property name="text">
        <string>create array</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>480</y>
      <width>273</width>
      <height>74</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>sentence:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="inputSentenceEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>560</y>
      <width>355</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>all sentences:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="resultSentencesEdit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1041</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
